mmv1/products/iamworkforcepool/api.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: IAMWorkforcePool
+display_name: Cloud IAM
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://iam.googleapis.com/v1/
+scopes:
+  - https://www.googleapis.com/auth/iam
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Identity and Access Management (IAM) API
+    url: https://console.cloud.google.com/apis/library/iam.googleapis.com/
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'targetLink'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: True
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+objects:
+  - !ruby/object:Api::Resource
+    name: 'WorkforcePool'
+    base_url: locations/{{location}}/workforcePools
+    self_link: locations/{{location}}/workforcePools/{{workforce_pool_id}}
+    create_url: locations/{{location}}/workforcePools?workforcePoolId={{workforce_pool_id}}
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      Represents a collection of external workforces. Provides namespaces for
```

Comment:
Do you want to add a note here on how a customer would gain access to the allowlist to be able to provision these resources? I assume customers may find this Terraform resource and try to provision it without knowing they need special permission to create them.

It may just be a link to GCP documentation on the feature, or something like that telling them to contact sales

Reply:
fixed